exercises/nucleotide-count/description.md

```diff
@@ -1,11 +1,19 @@
-Given a single stranded DNA string, compute how many times each nucleotide occurs in the string.
+Most people inherit not only their socioeconomic class from their parents, but also DNA (deoxyribonucleic acid), a set of chemical instructions that influence how their bodies are constructed!
+DNA is a long chain of other chemicals and the most important are the four nucleotides, Adenine, Cytosine, Guanine and Thymine. A single DNA chemical can contain billions of these four nucleotides and the order in which they occur is important!
```

Comment:
Maybe I should say molecule rather than chemical? Not sure which is clearer.

Reply:
It's not considered one molecule, is it? (I don't know anything about biology)

Reply:
It is a molecule :)

Reply:
> A molecule is an electrically neutral group of two or more atoms held together by chemical bonds.

DNA is held together by covalent and hydrogen bonds. Anyway, as someone who didn't know that, which is more confusing, saying "DNA molecule" or "DNA chemical"

Ah, I can just call it a "DNA chain" again instead.